.github/workflows/change.yaml

```diff
@@ -0,0 +1,29 @@
+name: Check for API breaks
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  check:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
+
+    - uses: actions/setup-python@v5
+      with:
+        python-version: "3.12"
+
+    - name: Check for API breaks
+      continue-on-error: true
```

Comment:
I can't remember what side effect this has anymore - does it mean that we get a green tick even if it fails? If so, we should make it loud and not a merge requirement imho. That way we can have the red tick and know that it was API breaking?

Reply:
Okay counteroffer, how about if the step fails (which indicates API breakage) a comment is added to the PR saying "This PR breaks API, be sure to add a note in the change log" I was thinking it would be annoying to have a red check if API breaks and you knew it was going to. That way the check will always be green, but it tells us in a more loud way then having to read a CI log file every time.

Reply:
To do that you'll need to trigger a separate workflow iirc. That can be a bit messy / time consuming. Could this be a follow-up PR?

Reply:
I don't think it needs to trigger a separate workflow. I just added a bit to do that, it worked in https://github.com/OpenFreeEnergy/openfe/pull/727/commits/192b5e95f7440b59cf955edeff6d22fea718c5d7 now I just need to add an API break to see if it does what we expect

Reply:
@mikemhenry is it not that anything that needs secrets (like comment posting) will fail for PRs coming from non-org members unless it's in a `pull_request_target` workflow type?